### PR TITLE
{bio}[GCC/11.2.0] bcl2fastq2 v2.20.0

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+preconfigopts += local_sed_libver % {'l': 'LIBXSLT', 'f': local_cmakelists}
+
+# remove hardcoded compilation flags
+local_cmakecxx = '%(builddir)s/bcl2fastq/src/cmake/cxxConfigure.cmake'
+preconfigopts += 'sed -i "s/-std=[a-z0-9\+]* //g;s/-O. //g" %s && ' % local_cmakecxx
+
+configopts = '-DBCL2FASTQ_VERSION:STRING=%(version)s '
+configopts += '-DBCL2FASTQ_PREFIX:STRING=%(installdir)s '
+configopts += '-DBCL2FASTQ_SOURCE_DIR:STRING=%(builddir)s/bcl2fastq/src '
+configopts += '-DBoost_NO_BOOST_CMAKE=ON '
```
Comment (akesandgren) [inline easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-GCC-11.2.0.eb:54]: Please keep the comment about this option from the recently merged PR that added it.

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-GCC-11.2.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-GCC-11.2.0.eb:54]: i.e. #14741

Comment (SethosII) [reply easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-GCC-11.2.0.eb:55]: @akesandgren I added the comment.

Comment (akesandgren) [reply easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-GCC-11.2.0.eb:55]: btw, do you mind fixing the typo I made? should be "use any system installed"

Comment (SethosII) [reply easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-GCC-11.2.0.eb:55]: @akesandgren No problem, I fixed the typo.

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by boegel** [comment]
broken tests are due to a bug in the easyconfigs test suite, which is being fixed in #14769

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by akesandgren** [comment]
Going in, thanks @SethosII!

**Pull request merged**